### DISCUSSION TITLE
Support runtime (Node.js) flags?

### DISCUSSION THREAD

**Discussion by chikamichi** [remix-run/remix#2191] [Proposals]
Hi,

Thank you for all the work on this toolset!

I stumbled upon a "niche" usecase: https://github.com/code-hike/codehike/discussions/142

tl;dr: it would have been nice Remix had "support" for --experimental-json-module, so as to be able to `import` .json files as modules from remix.config.js

But actually, it had me thinking: what if, instead of adding such a specific behavior to the code base, one was able to pass options/flags to the runtime instead?

**Comment by sergiodxa** [comment]
Just curios. Why you can’t already?
